server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -268,6 +270,20 @@ public void sendShuffleData(
                 + spd.getPartitionId()
                 + "]";
         try {
+          isShuffleIdDeleted =
```

Comment:
what will happen it if server  processes a new application?

Reply:
I think a new `appId` and `shuffleId` will be already registered during the `registerShuffle` phase, which means `isShuffleIdDeleted` will be false when a new application arrives.